spec/base_decorator_test_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rspec'
+require './base_decorator'
+
+describe BaseDecorator do
+  let(:nameable) { double('Nameable', correct_name: 'take name') }
+  subject { described_class.new(nameable) }
+
+  describe '#correct_name' do
+    it 'should return the correct name of the nameable object' do
+      expect(subject.correct_name).to eq('take name')
+    end
+  end
+end
```

Comment:
thank you bet it is running correctly with command `rspec spec`

Reply:
I merged the project because It says approved, and also it works fine for me with the correct command `rspec spec
![s2](https://user-images.githubusercontent.com/49729863/235164296-2014c166-b5f2-41e9-bfa2-2e9240b2fb84.PNG)
`

Reply:
![s2](https://user-images.githubusercontent.com/49729863/235164496-b61842c9-f8a0-4243-b917-260b94fcfc9e.PNG)

Reply:
> thank you bet it is running correctly with command `rspec spec`

I've also tried `rspec spec` but still gives the following error 
![Uploading Screenshot 2023-04-28 at 15.13.52.png…]()